docs/experiments-ltr-msmarco-document-reranking.md

```diff
@@ -43,16 +34,15 @@ Now, we have all things ready and can run inference:
 ```bash
 python -m pyserini.search.lucene.ltr \
   --index msmarco-doc-per-passage-ltr \
-  --queries collections/msmarco-ltr-document \
   --model collections/msmarco-ltr-document/msmarco-passage-ltr-mrr-v1 \
   --ibm-model collections/msmarco-ltr-document/ibm_model/ \
-  --data document \
+  --topic tools/topics-and-qrels/topics.msmarco-doc.dev.txt \
+  --qrel tools/topics-and-qrels/qrels.msmarco-doc.dev.txt \
   --output runs/run.ltr.msmarco-doc.tsv \
+  --data document \
```

Comment:
I was a bit confused by `--data` before. Is it just "document" vs. "passages"? If so, not sure what's a better name...  maybe `--granularity`?